api/v1/backupstoragelocation_types.go

```diff
@@ -117,31 +148,3 @@ const (
 
 // TODO(2.0): remove the AccessMode field from BackupStorageLocationStatus.
```

Comment:
should lines 149 and 150 also be removed?

Reply:
No, they have nothing to do with this PR, those are slated to be done for v2.0.